test/ci/buildkite/ngtf-cpu_ubuntu-bin-build.yaml

```diff
@@ -5,33 +5,36 @@
     label: ":gear: Setup"
     timeout_in_minutes: 30
     agents:
-    - "queue=cpu_gcc_4.8"
+    - "queue=cpu"
     parallelism: 1
 
   - wait
 
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
+      git clone https://github.com/tensorflow/tensorflow.git /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/tensorflow
+      ln -sf /localdisk/buildkite-agent/prebuilt_tensorflow_1_15_0/artifacts/tensorflow/tensorflow-1.15.0-cp36-cp36m-linux_x86_64.whl /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/tensorflow
```

Comment:
Can you please add some comments about these lines and why they are needed

Reply:
Done